modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -8,7 +8,7 @@ class FacilitiesInfoController < ApplicationController
       def index
         Rails.logger.info('Facilities info call start', { sort_method: params[:sort], user_uuid: @current_user.uuid })
         facility_ids = @current_user.va_treatment_facility_ids + @current_user.cerner_facility_ids
-        facilities = appointments_proxy.fetch_facilities_from_ids(facility_ids.uniq, true)
+        facilities = appointments_proxy.fetch_facilities_from_ids(facility_ids, true)
```

Comment:
Was this causing issues? I think the purpose of this was so we don't pull info multiple times for the same id.

Reply:
I moved the uniq to be within the fetch_facilities_from_ids method

Reply:
Ope, thanks. Didn't see that.